STEPS.md

```
@@ -3687,7 +3687,7 @@ Then the response should be a valid Atom feed
 
 >  Manage Drupal blocks.
 >  - Create and configure blocks with custom visibility conditions.
->  - Place blocks in regions and verify their rendering in the page.
+>  - Place blocks in regions and assert their configured region.
```

markdownlint-cli2 (0.23.2)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)